portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+    <body>
+        <header>
+        <div id ="header">
+            <h1 style="text-align:center"> My Projects </h1>
+            <a href="index.html"> home </a>
+        </div>
+        </header>
+        <div id="content">
+            <img src="http://g.recordit.co/HbRJhGXNiX.gif" alt="app example" width="200px" height="400px">
+            <span> This project was a warm-up for a workshop that gave training on making android apps.</span>
```

Comment:
Is there a way to have the description sit entirely to the right of the image, or at the bottom, instead of doing both - http://screen/qpbhtdv424Q?

Reply:
Siddharth, what do you think of taking this change as a follow-up issue instead of having it block thePR?

Depending on how large the change is (CSS styling can be tricky especially when you're trying to line things up...).

And I think it would be good to get this PR in -- there are lots of other changes in there.

Reply:
Yes I have already fixed this. But I made a new branch and I guess I accidentally pushed to this branch with this issue. should I just push my fixed solution and move to the other branch?

Reply:
Can you add the fix onto this branch? Under usual cases, we don't want to abandon pull requests -- the review comments on this are part of the history of the change. :) And it'll take the reviewers longer to review, because they'll lose state on what has already been reviewed.

If there's a specific commit on that you just need to bring over, then you can get the commit id from that branch (after seeing `git log`) and then `git cherry-pick $commit_id` on this branch. We can walk over how to do that in person tomorrow, if you like; cherry-pick is useful to know but there are some subtleties and pitfalls.

Otherwise, you can just copy over the change from that branch, and make a new commit.